src/main/resources/liquibase/crop_changelog/14_1_0.xml

```diff
@@ -0,0 +1,239 @@
+        </preConditions>
+        <comment>Drop Column cuid_bkp if exists from table changes.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE changes DROP COLUMN cuid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-14">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="germplsm" columnName="germuid_bkp"/>
+        </preConditions>
+        <comment>Drop Column germuid_bkp if exists from table germplsm.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE germplsm DROP COLUMN germuid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-15">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="locdes" columnName="duid_bkp"/>
+        </preConditions>
+        <comment>Drop Column duid_bkp if exists from table locdes.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE locdes DROP COLUMN duid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-16">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="methods" columnName="muid_bkp"/>
+        </preConditions>
+        <comment>Drop Column muid_bkp if exists from table methods.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE methods DROP COLUMN muid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-17">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="names" columnName="nuid_bkp"/>
+        </preConditions>
+        <comment>Drop Column nuid_bkp if exists from table names.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE names DROP COLUMN nuid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-18">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="udflds" columnName="fuid_bkp"/>
+        </preConditions>
+        <comment>Drop Column fuid_bkp if exists from table udflds.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE udflds DROP COLUMN fuid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-19">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="reflinks" columnName="refuid_bkp"/>
+        </preConditions>
+        <comment>Drop Column refuid_bkp if exists from table reflinks.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE reflinks DROP COLUMN refuid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-20">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="projectprop" columnName="projectprop_users_bkp"/>
```

Comment:
**projectprop_users_bkp** is a backup table, not column

Reply:
I apologize, I believe i miss read the document, i thought this are under the table projectprop and nd_geolocationprop. Although the tables projectprop_users_bkp and nd_geolocationprop_users_bkp doesn't exists in the current schema (i'm using maize crop)